lib_eio/fs.ml

```diff
@@ -12,6 +12,10 @@ exception Already_exists of path * exn
 exception Not_found of path * exn
 exception Permission_denied of path * exn
 
+class virtual read_at_offset = object
+  method virtual read_at_offset : int64 -> Cstruct.t -> int
```

Comment:
I don't particularly like the name, but POSIX and Lwt_unix both call this `pread`.

Elsewhere, we're using `Int63.t` as the type for offsets, to avoid an allocation.

Reply:
I renamed to `pread`, and switched to `Int63.t`.